Resources/Prototypes/Catalog/Fills/Boxes/general.yml

```diff
@@ -6,6 +6,8 @@
   components:
   - type: Sprite
     state: box
+  - type: StaticPrice
+    price: 1
```

Comment:
?

Reply:
Each carton is worth 10 units by default, with this I reduced to 1, made them more attractive for recycling than sales.